tcf_core/settings/base.py

```diff
@@ -174,80 +170,30 @@
     'social_core.pipeline.social_auth.load_extra_data',
     'social_core.pipeline.user.user_details',
 )
+SOCIAL_AUTH_USER_MODEL = 'tcf_website.User'
+
+AUTH_USER_MODEL = 'tcf_website.User'
 
 # Read-only access to Elastic
 ES_PUBLIC_API_KEY = env.str('ES_PUBLIC_API_KEY')
 ES_COURSE_SEARCH_ENDPOINT = env.str('ES_COURSE_SEARCH_ENDPOINT')
 ES_INSTRUCTOR_SEARCH_ENDPOINT = env.str('ES_INSTRUCTOR_SEARCH_ENDPOINT')
 
-# PROD SETTINGS
-if not DEBUG:
-
-    # Heroku configuration.
-    if env.bool("HEROKU", default=False):
-        import django_heroku
-        django_heroku.settings(locals())
-
-    # Gather information from environment variables.
-
-    # HOSTNAME = env.str('HOSTNAME')
-    # PUBLIC_IPV4 = env.str('PUBLIC_IPV4')
-
-    # SECURITY WARNING: App Engine's security features ensure that it is safe to
-    # have ALLOWED_HOSTS = ['*'] when the app is deployed. If you deploy a Django
-    # app not on App Engine, make sure to set an appropriate host here.
-    # See https://docs.djangoproject.com/en/1.10/ref/settings/ (from GCP
-    # documentation)
-    ALLOWED_HOSTS = ['*', 'thecourseforum.com']
+# Read-write access to Elastic
+ES_COURSE_DOCUMENTS_ENDPOINT = env.str('ES_COURSE_DOCUMENTS_ENDPOINT')
+ES_INSTRUCTOR_DOCUMENTS_ENDPOINT = env.str(
+    'ES_INSTRUCTOR_DOCUMENTS_ENDPOINT')
+ES_PRIVATE_API_KEY = env.str('ES_PRIVATE_API_KEY')
 
-    # if HOSTNAME:
-    #     ALLOWED_HOSTS.append(HOSTNAME)
-    # if PUBLIC_IPV4:
-    #     ALLOWED_HOSTS.append(PUBLIC_IPV4)
-
-    # Read-write access to Elastic
-    ES_COURSE_DOCUMENTS_ENDPOINT = env.str('ES_COURSE_DOCUMENTS_ENDPOINT')
-    ES_INSTRUCTOR_DOCUMENTS_ENDPOINT = env.str(
-        'ES_INSTRUCTOR_DOCUMENTS_ENDPOINT')
-    ES_PRIVATE_API_KEY = env.str('ES_PRIVATE_API_KEY')
-
-    DATABASES['default'] = {
-        'NAME': env.str('DB_NAME'),
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'USER': env.str('DB_USER'),
-        'PASSWORD': env.str('DB_PASSWORD'),
-        'HOST': env.str('DB_HOST'),
-        'PORT': env.str('DB_PORT'),
-        'OPTIONS': {'sslmode': 'require'},
-    }
-
-LOGGING = {
```

Comment:
Are you sure about just deleting logging like this?

Reply:
We've always debugged issues by reproducing them on Heroku/local and seeing the logs there (rather than on GAE), but `django_heroku` configures logging, as mentioned in the description.

Reply:
I thought `django_heroku` was just for Heroku - does it set up GAE logging? GAE logs (or rather the error logs, which I'm not sure is the same thing as this?) have been pretty helpful, so I think we should keep those if we're not already.

Reply:
Oh were they helpful? I thought no one was using it. I'll put it back.

Reply:
Done in 255a6169bf8ac84c20edbe8c929d8a7248700b29.